templates/template.rb

```diff
@@ -2,4 +2,9 @@ def run(s)
     # Your code goes here
 end
 
-puts run(ARGV[0])
+
+starting = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+answer = run(ARGV[0].lines)
```

Comment:
Breaks tpxp day 1 part 1. 
Blame @TPXP for unconsistent definition of `run()`

Reply:
that's fine, the template won't affect old submissions

Reply:
Most programs actually need this `.lines` call, so it's fine as well to put it here

Reply:
I would rather keep the call to `lines()` in `run`. It is consistent with other language templates where `run` receives a string as argument and not an array. Also the template should be general by default, and it is quite confusing to receive an array with a single element instead of a one-line string (for instance, the input of day-05 is a one-line string).

Let's keep the input parsing in `run`. Maybe something like `template.rb`:

```ruby
 def run(s)
    # Your code goes here
    a = s.lines
end

run(ARGV[0].lines)
```

Reply:
I wasn't sure about how other languages handled it. If the call to `.lines` is in `run`, let's leave it here (but since that's between the two timers anyway I don't think it changes anything to be honest).